tap-gui/configurator/api-docs.md

```diff
@@ -0,0 +1,602 @@
+
+# Package @tpb/plugin-catalog
+
+## Class EntityPageSurface
+
+Manipulate contents of pages for Backstage catalog entities.
+
+Each catalog entity subtype can specify a dedicated template. This surface standardizes the contents of those templates and allows for some amount of external configuration.
+
+Here's an example of adding a tab to the template for Backstage Service entities:
+
+```
+context.applyWithDependency(
+  AppRouteSurface,
```

Comment:
why use applyWithDependency with the AppRouteSurface here? doesn't seem like it's used

Reply:
Good question, not sure. Lifted and dropped from the codebase itself.

I'm inclined to keep it like this in case there's some weird order-of-operations issue or something that forces this call pattern I think?